packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -927,26 +876,27 @@ test(`zcfSeat.incrementBy, decrementBy, zcf.reallocate from zcf.makeEmptySeatKit
   });
 });
 
-test(`userSeat from zcf.makeEmptySeatKit - only these properties exist`, async t => {
-  const expectedMethods = [
-    'getProposal',
-    'getPayouts',
-    'getPayout',
-    'getOfferResult',
-    'hasExited',
-    'tryExit',
-    'getCurrentAllocationJig',
-    'getAllocationNotifierJig',
-    'getFinalAllocation',
-    'numWantsSatisfied',
-  ];
-  const { zcf } = await setupZCFTest();
-  const { userSeat: userSeatP } = zcf.makeEmptySeatKit();
-  const userSeat = await userSeatP;
-  // Note: these tests will fail if Zoe is actually in a different
-  // vat, since userSeat would be a presence.
-  t.deepEqual(Object.keys(userSeat), expectedMethods);
-});
+// this test fails because the userSeat is now a far instance
+test.failing(
+  `userSeat from zcf.makeEmptySeatKit - only these properties exist`,
```

Comment:
Is it worth keeping this test around at all?

Reply:
Looks gone. Resolving.